packages/apollo-react-relay-duct-tape/src/storeObservation/compiledHooks/useCompiledLazyLoadQuery.ts

```diff
@@ -104,13 +107,14 @@ export function useCompiledLazyLoadQuery(
     client,
     executionQueryDocument,
     variables,
+    options.fetchPolicy
   );
   // ...then fetch/watch data for only the calling component...
   const { data } = useApolloQuery(watchQueryDocument, {
     client,
     variables,
-    fetchPolicy: "cache-only",
     // ...but only once finished loading.
+    fetchPolicy: "cache-only",
```

Comment:
Nit: this moved line change isn't necessary

Reply:
Fixed it in the latest commit few minutes back. :)